### PR TITLE
refactor(utility): migrate locked and shorten function from TensorBaySDK

### PR DESCRIPTION
**Pull request by wangyuqing0424**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-27rqm5u TensorBay 代码迁移：utility 中通用的函数](https://app.clickup.com/t/27rqm5u)

File: graviti/utility/common.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+from threading import Lock
+from typing import Any, Callable, DefaultDict, Type, TypeVar
+
+_CallableWithoutReturnValue = TypeVar("_CallableWithoutReturnValue", bound=Callable[..., None])
+_T = TypeVar("_T")
+
+locks: DefaultDict[int, Lock] = defaultdict(Lock)
+
+
+def common_loads(object_class: Type[_T], contents: Any) -> _T:
```
Comment (Lee-000) [inline graviti/utility/common.py:19]: No need to migrate `common_loads` for now, it's part of the attr framework.

File: graviti/utility/__init__.py [hunk 2]
```diff
@@ -4,7 +4,9 @@
 #
 """Utility module."""
 
+
```
Comment (Lee-000) [inline graviti/utility/__init__.py:7]: Why the blank line?

File: graviti/utility/common.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
```
Comment (Lee-000) [inline graviti/utility/common.py:3]: The year is incorrect.

**Review by Lee-000 — APPROVED** [review]

**Review by linjiX — APPROVED** [review]

**Pull request merged**